client/components/back-button/index.tsx

```diff
@@ -4,7 +4,7 @@ import { useTranslate } from 'i18n-calypso';
 import './style.scss';
 
 type BackButtonProps = {
-	onClick?: () => void;
+	onClick?: ( event?: React.MouseEvent< HTMLButtonElement > ) => void;
 };
```

Comment:
This is the main change that goes outside the scope of the reader. The ReaderBackButton is a wrapper on this component, but must pass an onClick that uses the passed event.